workspace-grid@mathematical.coffee.gmail.com/extension.js

```diff
@@ -230,7 +232,7 @@ function indexToRowCol(index) {
  */
 function rowColToIndex(row, col) {
     // row-major. 0-based.
-    let idx = row * global.screen.workspace_grid.columns + col;
+    let idx = row * Utils.WS.getWS().workspace_grid.columns + col;
```

Comment:
Why not create a constant for the `Utils.WS.getWS()` workspace manager?

Reply:
This modification is inspired by what has been done in https://github.com/micheleg/dash-to-dock/commit/8398d41
I suppose we cannot cache the workspace manager (even if I am not sure it can really change... maybe, I don't know :) )